app/globals.css

```diff
@@ -73,7 +73,20 @@ h4 {
   @apply top-8;
 }
 
-.journey-map__container {
-  background-image: url("/images/map-bg.svg");
-  @apply bg-no-repeat bg-cover;
-}
\ No newline at end of file
+
+:root {
+  --light-background: #fafaff;
+  --dark-background: #111827;
+  --light-foreground: #020202;
+  --dark-foreground: #f9fafb;
+}
```

Comment:
I would have used the `primary` & `secondary` instead of `background` and `foreground`; Do we really need a primary +  background + foreground ?

Reply:
So like, we would simply invert primary with secondary and vice-versa for dark mode and light mode?

Reply:
But primary and foreground colors are different right? We can't have primary color for all texts.

Reply:
Change primary and secondary for light and dark;